### PR TITLE
fix agency dumping, turn it on by default

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

dumping the agency on shutdown was not working after the instance manager rewrite. Fixed by this PR.

- [x] :hankey: Bugfix

### CODE REVIEW

**Review by maierlars — APPROVED** [review]

File: js/client/modules/@arangodb/testutils/testing.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ let optionsDocumentation = [
 ];
 
 const optionsDefaults = {
-  'dumpAgencyOnError': false,
```
[the remark this answers is not in the document]
Comment (dothebart) [reply js/client/modules/@arangodb/testutils/testing.js:165]: yes, it should.

**Review by adityamukho — APPROVED** [review]
LGTM, although I'm not sure an agency dump would be of interest in all failure cases.

**Comment by maierlars** [comment]
> LGTM, although I'm not sure an agency dump would be of interest in all failure cases.

I'd rather like to have it, instead of missing it after the fact. It's not that expensive.

**Comment by dothebart** [comment]
In case of no errors its probably wiped again afterwards ;-)

**Pull request merged**